README.md

```diff
@@ -4,29 +4,22 @@
 # BCF REST API
 ![](https://raw.githubusercontent.com/BuildingSMART/BCF/master/Icons/BCFicon128.png)
 
-**Version 2.1** based on BCFv2.1.
+**Version 2.2** based on BCFv2.1.
 [BCF XML GitHub repository](https://github.com/BuildingSMART/BCF)
 
+[Version 1.0 of the Open CDE Foundation API specification is available here](https://github.com/BuildingSMART/shared-common-API/tree/v1.0)
```

Comment:
When rendered, these don't seem to have line breaks. Try adding extra linefeeds between these lines.

Reply:
Thanks.